bin/docs_docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM redocly/redoc AS redoc
+
+# Install packages
+RUN apk add --update --no-cache bash curl \
+    && rm -rf /var/cache/apk/*
```

Comment:
We can ignore `&& rm -rf /var/cache/apk/*` 
https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

Reply:
btw, do we really need to install `bash` and `curl` here?

Reply:
I guess its just leftover from previous solution - constantly pull docs from inside docker